components/lcd_touch/esp_lcd_touch_cst816s/idf_component.yml

```diff
@@ -1,8 +1,8 @@
-version: "1.0.2"
+version: "1.0.3"
 description: ESP LCD Touch CST816S - touch controller CST816S
 url: https://github.com/espressif/esp-bsp/tree/master/components/lcd_touch/esp_lcd_touch_cst816s
 dependencies:
   idf: ">=4.4.2"
   esp_lcd_touch:
-    version: "^1.0"
+    version: ">=1.0.4"
```

Comment:
Would `^1.0.4` be safer? In case we introduce a breaking change in esp_lcd_touch v2?

Reply:
Yes, you are right! Fixed.